client/dashboard/sites/emails/index.tsx

```diff
@@ -1,19 +1,205 @@
-import { siteBySlugQuery } from '@automattic/api-queries';
+import { Email, SiteDomain } from '@automattic/api-core';
+import { emailsQuery, siteBySlugQuery, siteDomainsQuery } from '@automattic/api-queries';
 import { useQuery } from '@tanstack/react-query';
-import { __ } from '@wordpress/i18n';
+import { useNavigate } from '@tanstack/react-router';
+import {
+	ExternalLink,
+	Icon,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
+import { __, sprintf } from '@wordpress/i18n';
+import { next, wordpress } from '@wordpress/icons';
+import { useMemo, useState } from 'react';
+import { purchasesRoute } from '../../app/router/me';
 import { siteRoute } from '../../app/router/sites';
+import { DataViewsCard } from '../../components/dataviews-card';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import GoogleLogo from './resources/google-logo';
+import type { View } from '@wordpress/dataviews';
+import type React from 'react';
+
+import './styles.scss';
+
+const fields = [
+	{
+		id: 'emailAddress',
+		label: __( 'Email Address' ),
+		enableGlobalSearch: true,
+		render: ( { item }: { item: Email } ) => {
+			let iconEl = <Icon icon={ wordpress } size={ 28 } className="professional-email-icon" />;
+			if ( item.type === 'forwarding' ) {
+				iconEl = <Icon icon={ next } size={ 28 } className="email-forwarder-icon" />;
+			}
+
+			if ( item.type === 'mailbox' && item.provider === 'google_workspace' ) {
+				iconEl = <GoogleLogo size={ 24 } className="google-workspace-email-icon" />;
+			}
+
+			return (
+				<HStack spacing={ 4 } justify="flex-start">
+					<div className="email-icon-wrapper">{ iconEl }</div>
+					{ item.type === 'mailbox' ? (
+						<ExternalLink href={ `https://mail.${ item.domainName }` }>
+							{ item.emailAddress }
+						</ExternalLink>
```

Comment:
Is this the actual mailbox URL or a fake one? I see references to `https://wp.titan.email` and `https://mail.google.com/mail/` in the code.

Also, according to the Figma design, seems we shouldn't display the underline style?

Figma | Current branch
--- | ---
<img width="226" height="41" alt="Screenshot 2025-09-22 at 10 54 55" src="https://github.com/user-attachments/assets/a640bac1-d79c-44ec-aeb1-d7977f18392b" /> | <img width="243" height="58" alt="Screenshot 2025-09-22 at 10 55 00" src="https://github.com/user-attachments/assets/a99f8c58-fc31-4745-99f3-56fa6e9b49f7" />

Reply:
Once we implement the fields, they should look the same: https://linear.app/a8c/issue/DOTDASH-474/hosting-dashboard-emails-implement-dataview-fields